### PR TITLE
Feature book comment list

### PR DESCRIPTION
**Pull request by eldani17**
## Summary

The comment component was created.
The Comment List component was created.

## Screenshots

test-cooment

#### Galaxy S (or similar)
image

#### Pixel 3XL (or similar)
image

## Trello Card

https://trello.com/c/TP7NJQgD/16-n-73-tabla-de-comentarios

### CODE REVIEW

File: wbooks/src/app/components/bookComment/styles.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  container: {
+    backgroundColor: white,
+    flex: 1,
+    flexDirection: 'row',
+    padding: 20
+  },
+  image: {
+    borderRadius: 50,
+    height: 50,
+    width: 50
```
Comment (mattgle) [inline wbooks/src/app/components/bookComment/styles.ts]: Extrae el 50 a una constante y lo usas.

File: wbooks/src/app/components/bookCommentList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const MINELEMENT = 2;
+const MAXELEMENT = 10;
+
+function BookCommentList({ commentList }: any) {
+  const [quantityElement, setQuantityElement] = useState(MINELEMENT);
+  let commentListFilter: any[] = [];
+
+  const filterComments = () => {
+    return commentList.slice(0, quantityElement);
+  };
```
Comment (mattgle) [inline wbooks/src/app/components/bookCommentList/index.tsx]:
```suggestion
  const filterComments = () => commentList.slice(0, quantityElement);
```

File: wbooks/src/app/components/bookCommentList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import BookComment from '../bookComment';
+
+import styles from './styles';
+
+const MINELEMENT = 2;
+const MAXELEMENT = 10;
+
+function BookCommentList({ commentList }: any) {
+  const [quantityElement, setQuantityElement] = useState(MINELEMENT);
+  let commentListFilter: any[] = [];
```
Comment (mattgle) [inline wbooks/src/app/components/bookCommentList/index.tsx]: Cambia el any por el type que vaya.

Comment (mattgle) [inline wbooks/src/app/components/bookCommentList/index.tsx]: Por que mejor no usas un setState para esto en vez de un let?

File: wbooks/src/app/components/bookCommentList/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  const renderItem: ListRenderItem<ICommentBook> = ({ item }) => {
+    const { text, author, url } = item;
+    return <BookComment text={text} url={url} author={author} />;
+  };
+
+  const itemSeparator = () => <View style={styles.separator} />;
+
+  const handleShowComment = () => {
+    setQuantityElement(quantityElement === MINELEMENT ? MAXELEMENT : MINELEMENT);
+    commentListFilter = filterComments();
```
Comment (mattgle) [inline wbooks/src/app/components/bookCommentList/index.tsx]: creo que usar setState es mejor aca. Probalo

**Review by mattgle — CHANGES_REQUESTED** [review]
Buen trabajo, te dejo algunos comments!

File: wbooks/src/app/components/bookComment/styles.ts (hunk 1 above)
Comment (eldani17) [reply wbooks/src/app/components/bookComment/styles.ts]: the variable was extracted to a constant

File: wbooks/src/app/components/bookCommentList/index.tsx (hunk 2 above)
Comment (eldani17) [reply wbooks/src/app/components/bookCommentList/index.tsx]: The declaration form of the arrow function was changed

File: wbooks/src/app/components/bookCommentList/index.tsx (hunk 3 above)
Comment (eldani17) [reply wbooks/src/app/components/bookCommentList/index.tsx]: the any type was changed to CommentBook []

Comment (eldani17) [reply wbooks/src/app/components/bookCommentList/index.tsx]: The logic of the implementation is changed. Everything is solved with two useStates.

File: wbooks/src/app/components/bookCommentList/index.tsx (hunk 4 above)
Comment (eldani17) [reply wbooks/src/app/components/bookCommentList/index.tsx]: The logic of the implementation is changed. Everything is solved with two useStates.

**Review by mattgle — APPROVED** [review]
LGTM!

**Pull request merged**